packages/cactus-test-tooling/src/test/typescript/integration/substrate/substrate-test-ledger-constructor.test.ts

```diff
@@ -0,0 +1,43 @@
+import test, { Test } from "tape-promise/tape";
+import { LogLevelDesc } from "@hyperledger/cactus-common";
+import { SubstrateTestLedger } from "../../../../main/typescript/substrate-test-ledger/substrate-test-ledger";
+import { pruneDockerAllIfGithubAction } from "../../../../main/typescript/github-actions/prune-docker-all-if-github-action";
+
+const testCase = "Instantiate plugin";
+const logLevel: LogLevelDesc = "TRACE";
+
+test("BEFORE " + testCase, async (t: Test) => {
+  const pruning = pruneDockerAllIfGithubAction({ logLevel });
+  await t.doesNotReject(pruning, "Pruning didn't throw OK");
+  t.end();
+});
+
+test(testCase, async (t: Test) => {
+  const options = {
+    publishAllPorts: true,
+    logLevel: logLevel,
+    imageName: "ghcr.io/rafaelapb/cactus-substrate-aio",
```

Comment:
@RafaelAPB @CatarinaPedreira After you updated the default image in the test ledger class specifying the image tags here will no longer be necessary and I recommend either removing it here or setting it to the same value that will be the new default)